doc/collate1d.rst

```diff
@@ -190,37 +190,140 @@ Archiving
-+-----------------------------------------+---------------+---+----------------------+---+
-|J132401.95+271237.80_DEIMOS_20130409.fits|      sbzk_1796|...|DE.20130409.22509.fits|...|
-+-----------------------------------------+---------------+---+----------------------+---+
-
++++++++++++++++++
+The ``by_object.dat`` file contains metadata about the coadded output files.
+The file is organized by the output file name, and has multiple rows per output 
+file: one row per extracted spectra that was coadded to create the file. Below 
+is the description of some of the columns in the ``by_id.dat`` file. All filenames
+are relative to the top level of the archive directory.
+
++-----------------+-----------------------------------------------------------+
+| Column Name     | Description                                               |
++=================+===========================================================+
+| filename        | The filename of the coadded output file.                  |
++-----------------+-----------------------------------------------------------+
+| maskdef_objname | The name of the object being coadded.                     |
++-----------------+-----------------------------------------------------------+
+| maskdef_id      | The slit id for the according to the mask definition.     |
++-----------------+-----------------------------------------------------------+
+| det             | The detector the spectrum was captured on.                |
++-----------------+-----------------------------------------------------------+
+| ra              | The RA of the source object, determined from the mask     |
+|                 | definition.                                               |
++-----------------+-----------------------------------------------------------+
+| dec             | The DEC of the source object, determined from the mask    |
+|                 | definition.                                               |
++-----------------+-----------------------------------------------------------+
+| med_s2n         | The signal to noise ratio of the extracted object.        |
++-----------------+-----------------------------------------------------------+
+| wav_rms         | The RMS in pixels of the wavelength solution.             |
++-----------------+-----------------------------------------------------------+
+| source_id       | The KOAID of the original source image.                   |
++-----------------+-----------------------------------------------------------+
+| spec1d_filename | The name of the spec1d file containing the spectrum.      |
++-----------------+-----------------------------------------------------------+
+| dispname        | The grating used for the source image.                    |
++-----------------+-----------------------------------------------------------+
+| decker          | The slitmask used for the source image.                   |
```

Comment:
I have a nomenclature complaint. I would prefer that the slit mask name be called mask name or slit mask name and not decker. A decker is something weirdly specific and DEIMOS does not have one.